shenyu-register-center/shenyu-register-common/src/main/java/org/apache/shenyu/register/common/config/ShenyuDiscoveryConfig.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shenyu.register.common.config;
+
+import java.util.Properties;
+
+/**
+ * The type Discovery config.
+ */
+public class ShenyuDiscoveryConfig extends PropertiesConfig {
+    private String name;
+
+    private String type;
+
+    private String serverList;
+
+    private Properties props = new Properties();
```

Comment:
PropertiesConfig has Properties props  no need to add agin

Reply:
Thank you for your suggestion. I will improve it